src/presets/netlify.ts

```diff
@@ -125,9 +126,10 @@ async function writeHeaders(nitro: Nitro) {
   const headersPath = join(nitro.options.output.publicDir, "_headers");
   let contents = "";
 
-  for (const [path, routeRules] of Object.entries(
-    nitro.options.routeRules
-  ).filter(([_, routeRules]) => routeRules.headers)) {
+  const rules = Object.entries(nitro.options.routeRules)
+    .sort((a, b) => b[0].split("/").length - a[0].split("/").length)
```

Comment:
Small issue with sorting `/test/a` is more specific than `/test/*` but if * is before, it remains before and shadows.

Reply:
nice catch - thank you ❤️